Public/Src/Cache/ContentStore/Distributed/NuCache/NuCacheCollectionUtilities.cs

```diff
@@ -237,29 +237,41 @@ public static IEnumerable<T> SkipOptimized<T>(this IReadOnlyList<T> original, in
         /// process, adding them to the same priority queue. This process repeats until the original enumerable has no
         /// more elements.
         /// </summary>
-        public static IEnumerable<T> QueryAndOrderInPages<T>(this IEnumerable<T> original, int pageSize, Comparer<T> comparer, Func<List<T>, IEnumerable<T>> query)
+        public static IEnumerable<T> QueryAndOrderInPages<T>(this IEnumerable<T> original, int pageSize, Comparer<T> comparer, Func<List<T>, IEnumerable<T>> query, float? takeoutFraction = null, int poolMultiplier = 2)

```

Comment:
I expect pool multiplier to be more like 8-16. It's great that it's configurable, but can our defaults be more in line with expectations? Or is that a bad default?